eventdata/challenges/frozen.json

```diff
@@ -0,0 +1,122 @@
+{% set p_pre_filter_shard_size = (pre_filter_shard_size | default(1)) %}
+{% set p_query_iterations = (query_iterations | default(20)) %}
+
+{
+  "name": "frozen-data-generation",
+  "description": "Index approximately 50GB of data into an index with with 2 primary shards. IDs are autogenerated by Elasticsearch, meaning there are no conflicts.",
+  "meta": {
+    "client_count": 8,
+    "benchmark_type": "frozen-data-generation"
+  },
+  "schedule": [
+    {
+      "name": "deleteindex_elasticlogs",
+      "operation": {
+        "operation-type": "delete-index",
+        "index": "elasticlogs"
+      }
+    },
+    {
+      "name": "create_elasticlogs",
```

Comment:
Nit: If the delete operation is called `deleteindex_elasticlogs` then this one should be called `createindex_elasticlogs` for consistency?

Reply:
Agreed